pkgs/applications/science/physics/crystfel/default.nix

```diff
@@ -145,10 +149,17 @@ let
     src = fetchFromGitHub {
       owner = "nexusformat";
       repo = pname;
-      rev = "d469f175e5273c1d488e71a6134f84088f57d39c";
-      sha256 = "1jrzzh75i68ad1yrim7s1nx9wy0s49ghkziahs71mm5azprm6gh9";
+      rev = "master";
```

Comment:
This is not reproducible. You need to use the hash of the current head commit of the branch

Reply:
You're right, of course. Fixed.